### ISSUE TITLE
discord.errors.Forbidden: 403 Forbidden (error code: 50001): Missing Acess

### ISSUE THREAD

**Issue by Afnan998782** [therealOri/noter#2]
I am having this issue for too much time and its not solving

**Comment by therealOri** [comment]
The "discord.errors.Forbidden: 403 Forbidden (error code: 50001): Missing Access" error message seems to occur when a user or bot account tries to perform an action for which it does not have sufficient permissions or access.

Likely In this case, the error message specifically indicates that the bot is missing access to perform a certain action. This can happen, if the bot is not properly authorized to perform the action in the specific server or channel where it is trying to execute the command.

To fix this error, you should check the permissions and access settings for the bot in the Discord server, and make sure that it has the necessary permissions to perform the action it is attempting. You may also need to double-check the bot's authentication credentials/token and make sure that it is properly authorized to access the server and channel where it is trying to perform the action.


Basically, tldr; make sure your token is good and that the bot has the correct permissions/access. (This code is also likely a bit older and discord mat have changed things or the the API wrapper I use may have changed stuff.)

**Comment by therealOri** [comment]
Since there has been no activity here for so long, I will assume that this has helped and I'll be marking this as closed.
